awscli/s3transfer/crt.py

```diff
@@ -840,7 +841,19 @@ def _default_get_make_request_args(
                 x.title() for x in request_type.split('_')
             )
 
-        if is_s3express_bucket(call_args.bucket):
+        arn_handler = _S3ArnParamHandler()
+        if (
+            accesspoint_arn_details := arn_handler.handle_arn(call_args.bucket)
```

Comment:
```suggestion
            (accesspoint_arn_details := arn_handler.handle_arn(call_args.bucket))
```

Without the parentheses to scope it, this failed for me with:

```
cannot access local variable 'accesspoint_arn_details' where it is not associated with a value
```